tests-wtr/dom-actions.test.html

```diff
@@ -0,0 +1,20 @@
+<html>
+<body>
+<script type="module">
+  import {runTests} from '@web/test-runner-mocha';
+
+  runTests(async () => {
+    await import('./dom-actions.test')
```

Comment:
not sure about the naming - currently it's dom-actions.test.html + dom-actions.test.ts, we might want to come up with something better

Reply:
Good idea! I really like that each DomActions method can have its own dedicated html file, previously I imagined all would be tested in a single one - this will be much cleaner and more maintainable.

I moved the files like so:

```diff
-dom-actions.test.html
-dom-actions.test.ts
+dom-actions/dom-actions.click.html
+dom-actions/dom-actions.click.ts
```

Let me know if that makes sense to you :)